scrapers/ManyVids.yml

```diff
@@ -19,4 +29,69 @@ xPathScrapers:
         Name: $details//h4/a/text()
       Image: //div[@id="rmpPlayer"]/@data-video-screenshot
 
-# Last Updated May 15, 2020
+  performerSearch:
+    performer:
+      Name: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/text()
+      URL:
+        selector: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/@href
+        replace:
+          - regex: '/(?:Store/Videos/)?$'
+            with: /About/
+
+  performerScraper:
+    common:
+      $aboutlabel: //*[@class='mv-about__container__details__list-unit']/span[@class='mv-about__container__details__list-label']
+    performer:
+      Name: //*[@class='mv-about__banner-name ']/text()
+      Gender:
+        selector: $aboutlabel[contains(text(), 'Gender')]/following-sibling::*
+        replace:
+          - regex: Trans
+            with: transgender_female
+      URL: //a[@data-tab='about']/@href
+      Twitter: //*[@class='mv-about__banner-social']/a/i[@class='icon twitter-icon']/../@href
+      Instagram: //*[@class='mv-about__banner-social']/a/i[@class='icon instagram-icon']/../@href
+      Birthdate:
+        selector: $aboutlabel[contains(text(), 'Age')]/following-sibling::*
+        replace:
+          - regex: ' \(.*\)$'
+            with: ''
+        parseDate: 2006.01.02
+      Ethnicity:
+        selector: $aboutlabel[contains(text(), 'Ethnicity')]/following-sibling::*
+      Country:
+        selector: $aboutlabel[contains(text(), 'Nationality')]/following-sibling::*
+      EyeColor:
+        selector: $aboutlabel[contains(text(), 'Eye Color')]/following-sibling::*
+      Height:
+        selector: $aboutlabel[contains(text(), 'Height')]/following-sibling::*
+        replace:
+          - regex: '.*?or '
+            with: ''
+          - regex: ' cm$'
+            with: ''
+          - regex: \D+[\s\S]+
+            with: ""
+      Measurements:
+        selector: $aboutlabel[contains(text(), 'Measurements')]/following-sibling::* | $aboutlabel[contains(text(), 'Breast Size')]/following-sibling::*
+      FakeTits:
+        selector: $aboutlabel[contains(text(), 'Breast Size')]/following-sibling::*[contains(text(), 'Cohesive Gel') or contains(text(), 'Silicon') or contains(text(), 'Saline')]
+        replace:
+          - regex: '\s+?[0-9]+.*'
+            with: ''
```

Comment:
since our field is `FakeTits: ` should we replace `Natural` with `No` and either replace everything else with "Yes" or leave it as it is ? eg `Silicon`

Reply:
Agreed that `Natural` should be picked up and replaced with `No`.
IDK how this field is currently handled, but I would like to retain as much information as possible, so for non-`Natural` values, I'd like to keep it as is (eg `Silicon`)

So for queries for performers `FakeTits` you would list all performers with non-empty, non-`No` `FakeTits`

Reply:
We have filters that have `equals`, `not equals`, `is null`, `is not null` i think so keeping `no` for `Natural` and filtering by `equals No` or `not equals No`  would work  and we can keep the extra info you want.